sunpy/map/mapbase.py

```diff
@@ -1261,9 +1255,10 @@ def plot(self, gamma=None, annotate=True, axes=None, **imshow_args):
 
         # Normal plot
         if annotate:
-            axes.set_title("{name} {date:{tmf}}".format(name=self.name,
-                                                        date=parse_time(self.date),
-                                                        tmf=TIME_FORMAT))
+            if title is True:
+                axes.set_title(self.name)
+            else:
+                axes.set_title(title)
```

Comment:
this will do bad things if title was `False`.

A better conditional would be:

```python
if title is True:
    title = self.name

if title:
    axes.set_title(title)
```